rfc/rfc002-authentication-token.md

```diff
@@ -71,10 +71,12 @@ A verifier must be able to verify the token without contacting external services
 ### 4.4. User identification
 
 The token must hold the correct attributes on the user identity for the specific scope the token is to be used for.
+This data is shared with a different legal context (eg: different organization) and is therefore subject to legislation.
+A citizen service number or social security number for example can not be used.
```

Comment:
Would it be useful to give an example of something that _can_ be used?

Reply:
_it depends_

Reply:
The IRMA/UZI specifications in `## 7. Supported means` provide useful examples?

Reply:
I don't think this is the place to add to much examples. I can remove the negative example to prevent confusion.